assets/src/edit-story/elements/media/index.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+export { default as getMediaSizePositionProps } from './getMediaSizePositionProps';
+export { default as getFocalFromOffset } from './getFocalFromOffset';
+export { default as EditPanMovable } from './editPanMovable';
+export { default as ScalePanel } from './scalePanel';
+
+export const CropBox = styled.div`
+  width: 100%;
+  height: 100%;
+  position: relative;
+  overflow: hidden;
+
+  &::after {
+    content: '';
+    display: block;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    border: 1px solid ${({ theme }) => theme.colors.mg.v1}70;
```

Comment:
I know this was just copy pasted, but what's with the 70 here? Opacity? We should probably use `rgba` for that.

Reply:
Hmm, seems like a typo to me, removed it.